src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -170,6 +169,7 @@ object Read extends Logging {
   def fromSAMRecordOpt(record: SAMRecord,
```

Comment:
Here it means it returns an Option - I think @ryan-williams introduced the convention - 

@ryan-williams this is something you had thought was pretty standard?

It returns `None` right now for `MappedRead` without an MDTag, which I think @timodonnell wants to address